src/positioner/test/index.js

```diff
@@ -112,3 +112,15 @@ test('It pushes the rect to the right if overflowing on the left side', t => {
     })
   )
 })
+
+test('It pushes the rect to the top if overflowing on the bottom side', t => {
```

Comment:
Perhaps this should merge with the test of the same type (I think it just didn't test for width)? https://github.com/segmentio/evergreen/blob/f7a15e97babcbcec118a2a8be6a314ae7cd9a65b/src/positioner/test/index.js#L92

Reply:
Ah good spot - thanks! Your unit test I think is still valid, I guess the unit test added was to test that the container is "locked" to the bottom of the view port, whereby the target is only _just_ off the view port. Updated the test.